cypress/support/commands.js

```diff
@@ -24,26 +24,28 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
-Cypress.Commands.add('login', (email, username, password) => {
-  cy.request('POST', '/api/users', {
-    user: {
-      email,
-      username,
-      password
-    }
-  }).then(response => {
-    const user = {
-      bio: response.body.user.bio,
-      effectiveImage: 'https://static.productionready.io/images/smiley-cyrus.jpg',
-      email: response.body.user.email,
-      image: response.body.user.image,
-      token: response.body.user.token,
-      username: response.body.user.username
-    };
-    window.localStorage.setItem('user', JSON.stringify(user));
-    cy.setCookie('auth', response.body.user.token);
-  });
-});
+Cypress.Commands
+  .add('login',
+    (email = 'gitepi3626@vaband.com', username, password = 'pXTDbz8429') => {
+      cy.request('POST', 'https://conduit.mate.academy/api/users/login', {
+        user: {
+          email,
+          username,
```

Comment:
You have a login endpoint. It means your request body should contain only email and password keys. You can remove username